c7n/resources/s3.py

```diff
@@ -495,6 +491,11 @@ def modify_bucket_tags(session_factory, buckets, add_tags=(), remove_tags=()):
             continue
 
 
+def get_region(b):
+    return b.get('Location', {
+        'LocationConstraint': 'us-east-1'})['LocationConstraint'] or 'us-east-1'
```

Comment:
there's another oddity that should be accounted for here, namely for a value of 'EU' it should translate to 'eu-west-1'

Reply:
we all love you aws 👍